ui/start.html

```diff
@@ -4,7 +4,8 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="flag-icon-css/css/flag-icon.css">
     <link rel="stylesheet" href="css/fonts.css">
-    <link rel="stylesheet" href="css/main.css">
+    <link rel="stylesheet" href="css/main.css" media="screen and (max-height: 800px)"> 
```

Comment:
This could be annoying to testers and developers, I'm open to suggestions on a better way to do it! I'll do some tests setting the resolution change to 1080 instead of 800, but I was having some issues earlier.